srcpkgs/python3-pyliblo3/patches/Cython 3.1.2.patch

```diff
@@ -0,0 +1,40 @@
+From 6b53ac0f1768f5f2f017b7011030ef6461366639 Mon Sep 17 00:00:00 2001
+From: Muhammad Alfi Syahrin <malfisya.dev@hotmail.com>
+Date: Tue, 17 Jun 2025 19:58:19 +0700
+Subject: [PATCH] Fix compilation with Cython 3.1.2
+
+---
+ pyliblo3/_liblo.pyx | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/pyliblo3/_liblo.pyx b/pyliblo3/_liblo.pyx
+index 4fa70df..dcc4da9 100644
+--- a/pyliblo3/_liblo.pyx
++++ b/pyliblo3/_liblo.pyx
+@@ -971,7 +971,7 @@ cdef class Message:
+         if t == 'i':
+             lo_message_add_int32(self._message, int(value))
+         elif t == 'h':
+-            lo_message_add_int64(self._message, long(value))
++            lo_message_add_int64(self._message, int(value))
+         elif t == 'f':
+             lo_message_add_float(self._message, float(value))
+         elif t == 'd':
+@@ -1012,7 +1012,7 @@ cdef class Message:
+             lo_message_add_true(self._message)
+         elif value is False:
+             lo_message_add_false(self._message)
+-        elif isinstance(value, (int, long)):
++        elif isinstance(value, (int)):
+             try:
+                 lo_message_add_int32(self._message, <int32_t>value)
+             except OverflowError:
+@@ -1067,7 +1067,7 @@ cdef class Bundle:
+ 
+         if len(messages) and not isinstance(messages[0], Message):
+             t = messages[0]
+-            if isinstance(t, (float, int, long)):
++            if isinstance(t, (float, int)):
+                 tt = _double_to_timetag(t)
+             elif isinstance(t, tuple) and len(t) == 2:
+                 tt.sec, tt.frac = t
```

Comment:
This hasn't been accepted upstream. We need to wait on their decision before pulling this patch.